src/app/client/src/app/modules/dashboard/components/re-issue-certificate/re-issue-certificate.component.spec.ts

```diff
@@ -459,4 +459,28 @@ describe('ReIssueCertificateComponent', () => {
     expect(component.unsubscribe$.next).toHaveBeenCalled();
     expect(component.unsubscribe$.complete).toHaveBeenCalled();
   });
+
+  it ('should return TRUE when the status is 2', () => {
```

Comment:
Update statement easy to understand.
'should enable "Re-Issue" button when status = 2 "